test/end-to-end/test_launcher_exits_on_supervisor_startup_failure.ps1

```diff
@@ -5,20 +5,20 @@
 
 Add-Type -TypeDefinition (Get-Content "$PSScriptroot/../../.expeditor/scripts/end_to_end/SupervisorRunner.cs" | Out-String)
 
-$env:TESTING_FS_ROOT = (Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName()))
-New-Item $env:TESTING_FS_ROOT -ItemType Directory -Force
+$env:FS_ROOT = (Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName()))
+New-Item $env:FS_ROOT -ItemType Directory -Force
 
-# Installing the launcher here because TESTING_FS_ROOT is an imperfect
+# Installing the launcher here because FS_ROOT is an imperfect
 # abstraction that needs to be removed. It turns out that even if
-# TESTING_FS_ROOT is in place, we still look for the launcher in
+# FS_ROOT is in place, we still look for the launcher in
 # `/hab` when we start up.
 #
-# Once we remove TESTING_FS_ROOT completely, we'll need to rethink how
+# Once we remove FS_ROOT completely, we'll need to rethink how
 # this test works, since we can't really make `/` read-only
```

Comment:
I'm not sure that this big chunk of exposition makes sense anymore; it should probably be removed... if this test still functions, that is :thinking:

Reply:
Fixed in (23ee7e3)[https://github.com/habitat-sh/habitat/pull/7642/commits/23ee7e3c29c1be30062bd429cbf8ca12892f873d]